checkbox-test-plans/units/cheese.pxu

```diff
@@ -0,0 +1,19 @@
+plugin: manual
+id: cheese
+flags: simple
+_description:
+ PURPOSE:
+       Check that cheese runs and works correctly
+ STEPS:
+      1. Run cheese from the desktop icon. Does the icon look correct? 
+      2. Make sure hamburger menu -> Help brings up the Cheese Help page
+      3. See if cheese connects to any available webcams
+      4. See if screenshots work
+      5. See if video recording works
```

Comment:
Check that it creates snapshots in the list of pictures and videos

Reply:
good idea